scripts/dream.py

```diff
@@ -108,6 +108,8 @@ def main():
 
     # preload the model
     gen.load_model()
+    #set additional option
+    gen.free_gpu_mem = opt.free_gpu_mem
```

Comment:
This is a bit awkward, but there's no other way to do it in the current model without adding more arguments to its __init__ method and then making the corresponding changes in dream.py.